FrameworkSystem/test/testLoggerWrapper/tests/TestLevels.py

```diff
@@ -0,0 +1,208 @@
+"""
+Test Levels
+"""
+
+__RCSID__ = "$Id$"
+
+# imports
+import unittest
+
+# sut
+from DIRAC import gLogger, oldgLogger
+from DIRAC.FrameworkSystem.test.testLoggerWrapper.tests.TestLoggerWrapper import TestLoggerWrapper
+
+
+class TestLevels(TestLoggerWrapper):
+  """
+  Test get and set levels.
+  """
+
+  def test_00shown(self):
+    """
+    Test the validity of the shown method
+    """
+    gLogger.setLevel('warn')
+    oldgLogger.setLevel('warn')
+
+    gLogger.debug('message')
+    oldgLogger.debug('message')
+
+    self.assertEqual(self.buffer.getvalue(), "")
+    self.assertEqual(gLogger.shown('debug'), False)
+    self.assertEqual(self.oldbuffer.getvalue(), "")
+    self.assertEqual(oldgLogger.shown('debug'), False)
+    self.buffer.truncate(0)
+    self.oldbuffer.truncate(0)
+    
+    gLogger.warn('message')
+    oldgLogger.warn('message')
+
+    self.assertIn("", self.buffer.getvalue())
+    self.assertEqual(gLogger.shown('warn'), True)
+    self.assertIn("", self.oldbuffer.getvalue())
+    self.assertEqual(oldgLogger.shown('warn'), True)
+    self.buffer.truncate(0)
+    self.oldbuffer.truncate(0)
+
+  def test_01setLevelGetLevel(self):
+    """
+    Set gLogger level to error and get it
+    """
+    gLogger.setLevel('error')
+    self.assertEqual(gLogger.getLevel(), 'ERROR')
+
+  def test_02setLevelCreateLog(self):
+    """
+    Set gLogger level to error and try to create debug and error logs
+    Differences between the two systems :
+    - gLogger: linear system level : difference between notice and error
+    - old gLogger: v system level : no difference between notice and error
+    """
+    gLogger.setLevel('error')
+
+    self.assertEqual(gLogger.shown('debug'), False)
+    self.assertEqual(gLogger.shown('verbose'), False)
+    self.assertEqual(gLogger.shown('info'), False)
+    self.assertEqual(gLogger.shown('warn'), False)
+    self.assertEqual(gLogger.shown('notice'), False)
+
+    self.assertEqual(gLogger.shown('error'), True)
+    self.assertEqual(gLogger.shown('always'), True)
+    self.assertEqual(gLogger.shown('fatal'), True)
+
+    oldgLogger.setLevel('error')
+
+    self.assertEqual(oldgLogger.shown('debug'), False)
+    self.assertEqual(oldgLogger.shown('verbose'), False)
+    self.assertEqual(oldgLogger.shown('info'), False)
+    self.assertEqual(oldgLogger.shown('warn'), False)
+    
+    self.assertEqual(oldgLogger.shown('notice'), True)
```

Comment:
In the old gLogger, 'notice' and 'error' are at the same level -30 and 30. So when I set the level to 'error', I have also 'notice' messages. 
In the new gLogger, as the level system is linear, notice is under error. So if I set the level to 'error' I have no 'notice' message. Do you understand ?